Cargo.toml

```diff
@@ -57,6 +59,7 @@ strip-ansi-escapes = "0.2.0"
 tap = "1.0.1"
 textwrap = { version = "0.16.0", features = ["terminal_size"] }
 tokio = { version = "1.28.2", features = ["full", "tracing"] }
+tokio-util = { version = "0.7.10", features = ["compat"] }
```

Comment:
The `smol` ecosystem has their own `async-compat` crate, but their `Compat` newtype wrapper thing doesn't implement `Clone` like `tokio-util`'s does, so I use `tokio-util` here.